ui/src/app/machines/components/HeaderStrip/ActionFormWrapper/utils/getProcessingLabel.js

```diff
@@ -0,0 +1,49 @@
+export const getProcessingLabel = (processingCount, totalCount, action) => {
```

Comment:
Seems a little weird to have a file named after a single function that it exports, maybe this should just be defined in the index?

Reply:
That's fair! I'll update it now